src/main/java/com/hackclub/hccore/listeners/AdvancementListener.java

```diff
@@ -7,7 +7,7 @@
 import org.bukkit.Statistic;
 import org.bukkit.advancement.Advancement;
 import org.bukkit.advancement.AdvancementProgress;
-import org.bukkit.craftbukkit.v1_16_R1.advancement.CraftAdvancement;
+import org.bukkit.craftbukkit.v1_16_R3.advancement.CraftAdvancement;
```

Comment:
Interesting this isn't `v1_16_R4`?

Reply:
Nope, this is the NMS version. See https://www.spigotmc.org/wiki/spigot-nms-and-minecraft-versions-1-16/